specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -396,6 +396,259 @@
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "privateEndpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private endpoint."
+          },
+          {
+            "name": "privateDnsZoneGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private dns zone group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "200": {
+            "description": "Delete successful."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Delete private dns zone group": {
+            "$ref": "./examples/PrivateEndpointDnsZoneGroupDelete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PrivateDnsZoneGroup"
+        ],
+        "operationId": "PrivateDnsZoneGroups_Get",
+        "description": "Gets the private dns zone group resource by specified private dns zone group name.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "privateEndpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private endpoint."
+          },
+          {
+            "name": "privateDnsZoneGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private dns zone group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Expands referenced resources."
```

Comment:
this section not needed.

Reply:
done, will push once I have resolved other comments.